clients/discord-bot/src/commands/administrator-commands/debug-commands.marshal.ts

```diff
@@ -91,25 +102,26 @@ export class DebugCommandsMarshal extends Marshal {
                     .setStyle("DANGER")
                     .setLabel("Throw Error on Message")
                     .setCustomId("throw error on message"),
-                options: {
-                },
-                instructions: "Send a message to throw an error within the Wizard Step.",
+                options: {},
+                instructions:
+                    "Send a message to throw an error within the Wizard Step.",
                 func: (r: Message): WizardFunctionOutput => {
                     throw new Error(r.toString());
                 },
             },
         ];
 
-        const baseInteractionHandler = async (interaction: MessageComponentInteraction): Promise<WizardFunctionOutput> => {
+        const baseInteractionHandler = async (
+            interaction: MessageComponentInteraction,
+        ): Promise<WizardFunctionOutput> => {
             await response.reactions.removeAll();
             // Acknowledge it
-            const opts = debugFunctions.find(df => df.customId === interaction.customId);
+            const opts = debugFunctions.find(
+                df => df.customId === interaction.customId,
+            );
```

Comment:
Also somewhat unsure about this; seems like this will bloat a lot of stuff

Reply:
Fixed https://github.com/SprocketBot/sprocket/pull/327/commits/973a46d75de2031f225fff5fab8f1953d63f4976